scripts/redshift/load_redshift.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from matrix.common.etl import run_etl, transform_bundle, finalizer_reload
+
+
+if __name__ == '__main__':
+    # All SS2 analysis bundles
+    query = {
+        "query": {
+            "bool": {
+                "must": [
+                    {
+                        "match": {
+                            "files.library_preparation_protocol_json.library_construction_approach.ontology": "EFO:0008931"
+                        }
+                    },
+                    {
+                        "match": {
+                            "files.donor_organism_json.biomaterial_core.ncbi_taxon_id": 9606
```

Comment:
Can we leave a comment as to why this query has a value of 9606?

Reply:
Yup, for context, 9606 is the ID for "Homo Sapiens"